xml/operations-maintenance-controller-pit_database_recovery.xml

```diff
@@ -8,132 +8,146 @@
+10.1.2.1         ardana002-cp1-c1-m1-guest ardana002-cp1-c1-m1-guest
+10.84.65.3       ardana002-cp1-c1-m1-EXTERNAL-VM ardana002-cp1-c1-m1-external-vm
+    </screen>
     <para>
-     Log in to the first host member from the <literal>FND-MDB</literal> group.
+     In this example, <literal>192.168.24.21</literal> is the IP address for
+     the host.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Become root:
+     SSH into the host.
     </para>
-<screen>sudo su</screen>
-   </listitem>
-   <listitem>
+<screen>ssh ardana@192.168.24.21</screen>
+   </step>
+   <step>
     <para>
-     Source the backup environment file
+     Source the backup file.
     </para>
-<screen>source /home/stack/backup.osrc</screen>
-   </listitem>
-   <listitem>
+<screen>source /var/lib/ardana/backup.osrc</screen>
+   </step>
+   <step>
+    <para>
+     Find the client id.
+    </para>
+<screen>
+<?dbsuse-fo font-size="0.65em"?>
+freezer client-list
++-----------------------------+----------------------------------+-----------------------------+-------------+
+| Client ID                   | uuid                             | hostname                    | description |
++-----------------------------+----------------------------------+-----------------------------+-------------+
+| ardana002-cp1-comp0001-mgmt | f4d9cfe0725145fb91aaf95c80831dd6 | ardana002-cp1-comp0001-mgmt |             |
+| ardana002-cp1-comp0002-mgmt | 55c93eb7d609467a8287f175a2275219 | ardana002-cp1-comp0002-mgmt |             |
+| ardana002-cp1-c0-m1-mgmt    | 50d26318e81a408e97d1b6639b9404b2 | ardana002-cp1-c0-m1-mgmt    |             |
+| ardana002-cp1-c1-m1-mgmt    | 78fe921473914bf6a802ad360c09d35b | ardana002-cp1-c1-m1-mgmt    |             |
+| ardana002-cp1-c1-m2-mgmt    | b2e9a4305c4b4272acf044e3f89d327f | ardana002-cp1-c1-m2-mgmt    |             |
+| ardana002-cp1-c1-m3-mgmt    | a3ceb80b8212425687dd11a92c8bc48e | ardana002-cp1-c1-m3-mgmt    |             |
++-----------------------------+----------------------------------+-----------------------------+-------------+</screen>
+    <para>
+     In this example, the client id of host
+     <literal>ardana002-cp1-c1-m1-mgmt</literal> is
+     <literal>ardana002-cp1-c1-m1-mgmt</literal>.
+    </para>
+   </step>
+   <step>
     <para>
      List the jobs
     </para>
-<screen>freezer-scheduler -c &lt;hostname&gt; job-list</screen>
-   </listitem>
-   <listitem>
+<screen>freezer job-list -C ardana002-cp1-c1-m1-mgmt</screen>
+   </step>
+   <step>
     <para>
-     Get the id corresponding to the job <literal>&lcm; Default: Mysql restore
+     Get the corresponding job id for <literal>Ardana Default: mysql restore
      from Swift</literal>.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Launch the restore.
+     Launch the restore process with <literal>freezer job-start</literal>
+     <replaceable>JOB-ID</replaceable>.
```

Comment:
Either

           Launch the restore process with  
           <command>freezer job-start <replaceable>JOB-ID</replaceable></command>.

or

          <para>Launch the restore process with:</para>
          <screen>freezer job-start <replaceable>JOB-ID</replaceable></screen>